docs/docs-ref-autogen/outlook/office.appointmentcompose.yml

```diff
@@ -595,6 +595,48 @@ items:
       level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)</td> <td>ReadItem</td>
       </tr> <tr> <td>[Applicable Outlook mode](https://docs.microsoft.com/outlook/add-ins/#extension-points)</td>
       <td>Appointment Organizer</td> </tr> </table>
+
+      #### Examples
+
+
+      ```javascript
+
+      var locationDetails = [ 
+          {
+              "id": "Contoso",
+              "type": Office.MailboxEnums.LocationType.Custom
+          }
+      ];
+
+      Office.context.mailbox.item.enhancedLocation.addAsync(locationDetails);
+
+      Office.context.mailbox.item.enhancedLocation.getAsync(callbackFunction);
```

Comment:
For some reason there's no blank line rendering in the published doc between the `addAsync` call and the `getAsync` call. Can you please add an extra blank line between these lines (in the snippet file, where this code originates)?